x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils.test.ts

```diff
@@ -667,12 +682,10 @@ describe.each([
       soClient.find.mockClear();
     });
 
-    test('returns original action if Elasticsearch query fails', async () => {
-      clients.core.savedObjects
-        .getClient()
-        .find.mockRejectedValueOnce(new Error('failed to query'));
+    test('returns error if Elasticsearch query fails', async () => {
+      soClient.find.mockRejectedValue(new Error('failed to query'));
       const result = await swapActionIds(mockAction, soClient);
-      expect(result).toEqual(mockAction);
+      expect((result as Error).message).toEqual('failed to query');
```

Comment:
Not sure if maybe this is a preferred pattern for these? Might need to tweak it a bit, but something like:

```suggestion
      await expect(() => swapActionIds(mockAction, soClient)).toThrowErrorMatchingInlineSnapshot('failed to query');
```

Reply:
Ah yeah I like that better. I'll update this!